alloc-aware-optional.html

```diff
@@ -703,7 +707,9 @@ <h2>Proposed wording</h2>
 object contains a value, val points to the contained value.
 </ins></blockquote>
 <blockquote><ins>Member <tt>M_pmr</tt> is provided for exposition only and points to the memory_resource used when
-creating the contained value.</tt>.
+creating the contained value. If the value type provides <tt>get_allocator()</tt> member function, the implementation is 
```

Comment:
Because the note says that, and we don't need to normatively say that an exposition-only member might not be actually there; that's what exposition-only already means.